src/graphics/graphics-device.js

```diff
@@ -253,11 +260,12 @@ class GraphicsDevice extends EventHandler {
     maxVolumeSize;
 
     /**
-     * The maximum supported texture anisotropy setting.
+     * The highest shader precision supported by this graphics device. Can be 'hiphp', 'mediump' or
+     * 'lowp'.
      *
-     * @type {number}
+     * @type {string}
```

Comment:
if this one wasn't public, perhaps hide this as well .. at least till we find some common interface between webgl/webgpu (if needed there)

Reply:
It was public. See here:

https://developer.playcanvas.com/api/pc.GraphicsDevice.html#precision